build/script/travis-build.sh

```diff
@@ -10,15 +10,24 @@ if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
   export DISPLAY=:99.0
   sh -e /etc/init.d/xvfb start
   sleep 3
+
+  # Install Neovim
+  curl -LO https://github.com/neovim/neovim/releases/download/v0.2.2/nvim.appimage
```

Comment:
Was this why the Linux builds were failing? (No Neovim instance?)

Reply:
Yes, exactly! Most of the ci tests rely on having Neovim around - since there wasn't an `nvim` to pick up, those would all fail.